client/python/samples/README.md

```diff
@@ -198,20 +212,21 @@ docker run -d --rm -v ${PWD}/models/vehicle-detection:/models/vehicle-detection
 ```
 
 
-### Detect vehicle  <a name="detect-vehicle">
+### Detect vehicle  <a name="detect-vehicle-grpc">
 
 
 #### **Make vehicle detection prediction using images in binary format:**
 
 - Command
 
 ```bash
-usage: vehicle_detection_predict_binary.py [-h] --images_dir IMAGES_DIR
+usage: vehicle_detection_predict_binary_grpc.py [-h] --images_dir IMAGES_DIR
                                            [--grpc_address GRPC_ADDRESS]
                                            [--grpc_port GRPC_PORT]
                                            [--model_name MODEL_NAME]
                                            [--model_version MODEL_VERSION]
-                                           --output_save_path OUTPUT_SAVE_PATH
+                                           [--output_save_path OUTPUT_SAVE_PATH]
```

Comment:
Can we change name of this param to `output_dir` (to be more similar to `images_dir`)